README.md

```diff
@@ -1 +1,17 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## A brief introduction of me
```

Comment:
Good practice is to add a line break between headers and text content. Will avoid formatting issues in the future